app/services/mapping/governmentagencygoodsitem/GovernmentAgencyGoodsItemBuilder.scala

```diff
@@ -19,15 +19,30 @@ package services.mapping.governmentagencygoodsitem
 import models.declaration.governmentagencygoodsitem.Formats._
 import models.declaration.governmentagencygoodsitem.GovernmentAgencyGoodsItem
 import services.ExportsItemsCacheIds
+import services.cache.ExportItem
 import uk.gov.hmrc.http.cache.client.CacheMap
+import wco.datamodel.wco.dec_dms._2.Declaration
 import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment.{
   GovernmentAgencyGoodsItem => WCOGovernmentAgencyGoodsItem
 }
-import wco.datamodel.wco.declaration_ds.dms._2.GovernmentAgencyGoodsItemStatisticalValueAmountType
 
 import scala.collection.JavaConverters._
 
 object GovernmentAgencyGoodsItemBuilder {
```

Comment:
Similar to above, can we make this a class and inject the dependent builders

Reply:
this is still TODO will fix when I get to that builder